PROJECT

```diff
@@ -14,14 +14,6 @@ resources:
   kind: BannedUser
   path: ../github.com/codeready-toolchain/api/api/v1alpha1
   version: v1alpha1
-- api:
-    crdVersion: v1
```

Comment:
is this file auto-generated?

Reply:
No, it's not auto-generated. It's a manual change when adding/removing a CRD. It's mentioned in the api repo's PR template: https://github.com/codeready-toolchain/api/blob/master/.github/pull_request_template.md